src/frontend/js/login.js

```diff
@@ -107,31 +100,32 @@ div:has(#submit_btn) {
       <h1 class="title">Come ti chiami?</h1>
       <div>
         <label>
-          Username: <br/>
-        <div class="text-container">
-          <input
-            type="text"
-            @input=${this.onUsernameInput}
-            .value=${this.username}
-          />
-        </div>
+          Username
+          <div class="text-container">
+            <input
+              type="text"
+              @input=${this.onUsernameInput}
+              .value=${this.username}
+              placeholder="Inserisci il nome utente"
+            />
+          </div>
         </label>
       </div>
 
       <div>
         <label>
-          Password: <br/>
+          Password
           <div class="text-container">
-          <input
-            type=${ this.pswVisibility ? 'text' : 'password' }
-            @input=${this.onPasswordInput}
-            .value=${this.password}
-          />
-          <span @click=${this.setVisibility}>toggle</span>
+            <input
+              type=${ this.pswVisibility ? 'text' : 'password' }
+              @input=${this.onPasswordInput}
+              .value=${this.password}
+              placeholder="Inserisci la password"
+            />
+            <span @click=${this.setVisibility} class="material-symbols-rounded"> ${this.pswVisibility ? "visibility" : "visibility_off"} </span>
```

Comment:
la classe CSS material-symbols-rounded non esiste in questo componente, crearla per applicare il font "Material Icons" per fare in modo che le icone vengano mostrate correttamente.
Attenzione all'operatore ternario: dovrebbe essere il contrario.